packages/lib-core/src/store/PluginLoader.ts

```diff
@@ -25,17 +25,19 @@ type PluginLoadResult =
 
 type PluginLoadListener = (pluginName: string, result: PluginLoadResult) => void;
 
+export const pluginManifestFile = 'plugin-manifest.json';
+export const remoteEntryScript = 'plugin-entry.js';
```

Comment:
We found that serving static assets from CDN is often unreliable and it serves them constantly from cache. For such a fast paced release that is really unwanted as you might depend on the newest version. We've found out that the name is better to be generated with date and `fullhash`
```JS
const remoteEntryScript = (useFileHash) => `plugin-entry${useFileHash ? `.${Date.now()}.[fullhash]` : ''}.js`
```

However this feels out of place, so it might be better to change this to `remoteEntryScript = 'plugin-entry'` and later add the `.js` or fullhash.

Reply:
Yup, dynamically generated entry script name brings some more complexity but I see the argument for it being a `.js` file and potentially cached forever by a CDN.